dev-libs/libqt5pas/libqt5pas-1.2.9.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+LAZRELEASE="2.2.4"
```

Comment:
MY_P="${P}-0"

Reply:
It is going to expand into something like libqt5pas-1.2.9-0, right? But they put the sources inside the lezarus-2.2.4...tar... I didn't test it, but I think it's going to be a problem.

Reply:
Let me check

Reply:
I confirm that it works

Reply:
Yeah, I meant ${P} wasn't going to work for libqt5pas. 
MY_P="lazarus-2.2.4-0" does work. 
Tested and committed.